### DISCUSSION TITLE
I don't know the arg return_contacts=True mean when inference.

### DISCUSSION THREAD

**Discussion by CNwangbin** [facebookresearch/esm#183] [Q&A] [answered]
Just this line:`results = model(batch_tokens, repr_layers=[33], return_contacts=True)`.
Does that mean the return representation could ues for some tasks about protein-contact when the flag is True?

**Comment by CNwangbin** [comment, marked as answer]
I'm clear about that now.The results is a dict.If the flag `return_contacts=True` ,the results dict will contain a key "contact",vice versa.
Is that right?

**Reply by tomsercu** [reply]
that's right

**Reply by CNwangbin** [reply]
Thanks.
